collectors/COLLECTORS.md

```diff
@@ -501,6 +501,40 @@ the `go.d.plugin`.
 These collectors are developed and maintained by third parties and, unlike the other collectors, are not installed by
 default. To use a third-party collector, visit their GitHub/documentation page and follow their installation procedures.
 
+<details>
+<summary>Typical third party Python collector installation instructions</summary>
+<br>
+In general the below steps should be sufficient to use a third party collector.
+
+1. Download collector code file into folder expected by Netdata.
```

Comment:
Hello @andrewm4894 ,

I suggest to add a link for "folder expected by Netdata". 

Best regards!

Reply:
I dont think we have a good candidate link for this now that i look.